files/image_config/bash/wrapper_logger.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Do we need two copies of the same file? Can we reference the same file for both the host OS and docker containers, or symlink these two paths so that there is only one copy?

Reply:
@theasianpianist docker builds cannot refer to file anything but inside docker/ dir. So the only option is to refer to docker/ file in `files/build_templates/sonic_debian_extension.j2` which IMO is not neat. Keeping this file indepenent for host and docker build looks to be necessary.